src/components/organisms/header/header.scss

```
@@ -0,0 +1,112 @@
+@use "../../variables" as *;
+
+.navbar {
+  background-color: $black;
+  width: auto;
+
+  @media screen and (min-width: $breakpoint-xl) {
+    background-color: $black_50;
+    width: auto;
+    padding: 0 30px;
+    height: 90px;
+
+    .nav-item.dropdown {
+      .dropdown-menu.concept-dropdown {
+        display: none;
+      }
+      &:hover {
+        .dropdown-menu.concept-dropdown {
+          display: block;
+        }
+      }
+    }
+    .plus-button {
+      display: none;
+    }
+  }
+
+  .collapse.navbar-collapse {
+    justify-content: flex-end;
+  }
+
+  .plus-button {
+    margin-right: 16px;
+  }
+
+  .navbar-toggler-icon {
+    background-image: url(../../../img/humberger-bar.svg);
+  }
+
+  .switch {
+    width: 120px;
+
+    @media screen and (min-width: $breakpoint-xl) {
+      width: 140px;
+    }
+  }
+  .navbar-brand {
+    margin-bottom: 12px;
+  }
+  .dropdown-menu.concept-dropdown {
+    padding: 0;
+    border: none;
+    margin: 0;
+  }
```

```suggestion
  .dropdown-menu.concept-dropdown {
    padding: 0;
    border: none;
    margin: 0;
    &.collapsing {
      display: block;
    }
  }
```
こんな感じで `collapsing` に対して `display` を設定することで＋ボタンを押したあとのタイムラグがなくなると思うのでやってみてください